src/components/pages/realmHome/index.tsx

```diff
@@ -166,6 +171,48 @@ const AutocompleteOrganizations = ({ realm, userStorage }: any) => {
 	);
 };
 
+interface FavoriteButtonProps {
+	realm: string;
+}
+
+const FavoriteButton = ({ realm }: FavoriteButtonProps) => {
+	const dispatch = useDispatch();
+	const favoriteRealm = useSelector(
```

Comment:
I think a favorite us should be included in the favorite realm. Otherwise the redirection can be misguiding